src/resources/js/events-admin.js

```diff
@@ -369,14 +369,18 @@ jQuery( document ).ready( function( $ ) {
 	};
 
 	var toggle_linked_post_fields = function( event ) {
-		var $select   = $( this ),
-			$group    = $select.closest( 'tbody' ),
-			$edit     = $group.find( '.edit-linked-post-link a' ),
-			edit_link = $edit.attr( 'data-admin-url' ),
-			choice    = 'undefined' === typeof event.added ? {} : event.added;
+		var $select   = $( this );
+		var	$group    = $select.closest( 'tbody' );
```

Comment:
Alignment is weird here. Also on the Edit and choice variables.

Reply:
Bleh, yes it is